jax/image/scale.py

```diff
@@ -165,10 +165,10 @@ def _resize(image, shape: Sequence[int], method: Union[str, ResizeMethod],
            f' {shape} vs {image.shape}')
     raise ValueError(msg)
   if isinstance(method, str):
-    method_id = ResizeMethod.from_string(method)
-  if method_id == ResizeMethod.NEAREST:
+    method = ResizeMethod.from_string(method)
+  if method == ResizeMethod.NEAREST:
     return _resize_nearest(image, shape)
-  kernel = _kernels[method_id]
+  kernel = _kernels[method]
```

Comment:
mypy appears to be unhappy with this. The best fix in this kind of situation is usually to add `assert isinstance(method, ResizeMethod)` above this line

Reply:
Ok, done!
I run `mypy` locally and seems to be fine now :-)

PS: for the record I got this other error from it
```python
jax/tree_util.py:217: error: Argument 1 to "dict" has incompatible type "Tuple[Tuple[Any, ...], Tuple[Any, ...]]"; expected "Iterable[Tuple[<nothing>, <nothing>]]"
```